docs/source/track_errors/ctc.rst

```diff
@@ -0,0 +1,192 @@
+            ys = [
+                matched.gt_graph.graph.nodes[u]["y"],
+                matched.pred_graph.graph.nodes[v]["y"],
+            ]
+            ax.plot(xs, ys, color=color, linestyle="dashed")
+
+
+    def plot_matched(examples, annotations, title):
+        gt_color = "black"
+        pred_color = "blue"
+        mapping_color = "grey"
+        fig, ax = plt.subplots(1, len(examples) + 1, figsize=(3 * len(examples) + 1, 2))
+        for i, (matched, anns) in enumerate(zip(examples, annotations)):
+            axis = ax[i]
+            xbounds, ybounds = plot_graph(axis, matched.gt_graph, color=gt_color, annotations=anns)
+            bounds = plot_graph(axis, matched.pred_graph, color=pred_color, annotations=anns)
+            xbounds.extend(bounds[0])
+            ybounds.extend(bounds[1])
+            plot_matching(axis, matched, color=mapping_color)
+            axis.set_ybound(min(ybounds) - 0.5, max(ybounds) + 0.5)
+            axis.set_xbound(min(xbounds) - 0.5, max(xbounds) + 0.5)
+            axis.set_ylabel("Y Value")
+            axis.set_xlabel("Time")
+
+        handles = [
+            Patch(color=gt_color),
+            Patch(color=pred_color),
+            Patch(color=mapping_color),
+            Patch(color='orange'),
+            Patch(color='purple')
+        ]
+        labels = ["Ground Truth", "Prediction", "Mapping", "Edge Annotations", "Node Annotations"]
+        ax[-1].legend(handles=handles, labels=labels, loc="center")
+        ax[-1].set_axis_off()
+        plt.tight_layout()
+        fig.suptitle(title, y=1.05)
+
+Nodes
+-----
+
+True Positives
+^^^^^^^^^^^^^^
+
+A true positive node is one that is matched to only one node in the predicted graph. Additionally, the predicted node is not matched to any other node in the ground truth. True positive nodes are annotated on both the ground truth and the predicted graph.
+
+False Positives
+^^^^^^^^^^^^^^^
+
+False positive nodes are annotated on the predicted graph and correspond to a predicted node without a match in the ground truth graph.
+
+False Negatives
+^^^^^^^^^^^^^^^
+
+False negative nodes are annotated on the ground truth graph and correspond to a ground truth node without a match in the predicted graph.
+
+Non-Split
+^^^^^^^^^
+
+Non-split nodes are annotated on the predicted graph and correspond to a node in the prediction that has been matched to two nodes in the ground truth graph.
+
+.. jupyter-execute::
+    :hide-code:
+
+    plot_matched([ex_graphs.two_to_one(t) for t in [0, 1]], [{4: "NS"}, {5: "NS"}], "Non-Split Nodes")
+
+
+Edges
+-----
+
+False Positives
+^^^^^^^^^^^^^^^
+
+False positive edges are annotated on the predicted graph. An edge is considered a false positive if both nodes are true positive nodes, but the edge does not match to any edge in the ground truth graph. In the example below, edge (4, 8) is a false positive.
+
+.. jupyter-execute::
+    :hide-code:
+
+    # plot_matched([ex_graphs.crossover_edge()], [{}], "")
```

Comment:
Looking at the readthedocs preview, this example doesn't show up, only the ones below it do. Possibly a syntax error here?

Reply:
I added this function on another branch (I think the track-errors branch). When I merge that set of PRs in I'll uncomment out this line so that the plot shows up on the docs.